src/main/java/edu/cornell/kfs/sys/service/impl/CuPostalCodeValidationServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package edu.cornell.kfs.sys.service.impl;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.KFSKeyConstants;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.service.impl.PostalCodeValidationServiceImpl;
+import org.kuali.rice.kns.datadictionary.validation.fieldlevel.ZipcodeValidationPattern;
+import org.kuali.rice.krad.util.GlobalVariables;
+import org.kuali.rice.location.api.state.State;
+import org.kuali.rice.location.api.state.StateService;
+
+public class CuPostalCodeValidationServiceImpl extends PostalCodeValidationServiceImpl {
```

Comment:
Since we already have an overlay to locally customize PostalCodeValidationServiceImpl, I would just modify that rather than creating a new class. We should look to contribute our customizations to this class back to base code.